my_class.rb

```diff
@@ -0,0 +1,17 @@
+require my_enumerable
+
+# Represents a list of elements that pass certain conditions
```

Comment:
Kindly require the `my_enumerable` file properly so that there are no errors when running the `my_class` file.

![image](https://github.com/0sugo/enumerables/assets/38088818/bc5be5fe-f625-4a3b-9877-662dc85e2969)

- This error is due to the missing quotations around the file name, also consider using `require_relative` instead of `require`. You can read more about the difference between require and require_relative [here](https://www.akshaykhot.com/difference-between-load-autoload-require-in-ruby/#:~:text=Use%20require%20when%20you%20want,to%20the%20current%20working%20directory.).

Your statement should be `require_relative 'my_enumerable'`  instead of  `require my_enumerable`.

Otherwise, all the methods tests pass great job!!! 💯 

![image](https://github.com/0sugo/enumerables/assets/38088818/d5780dc2-ac3c-491e-a091-b27f6a8a0b54)

Reply:
Thanks so much @KDlamini  for the correction.We have fixed that